### ISSUE TITLE
sails.request

### ISSUE THREAD

**Issue by BraunreutherA** [balderdashy/sails#3388]
Hey there,

I was heavily using sails.request for integration tests as it is a lot faster then using something like superset cause it doesn't need a http server and we can use sails.load instead of lift in the bootstrap file. 
Now I got to a problem when I wanted to use skipper for file uploading. It seams that some middleware like skipper are not used on sails.request, even if if use sails.lift - For the moment I'm using superset as a workaround. Is it possible to fix this or isn't it possible to apply all middleware to the req when using sails.request method? 

Best,
Alex

**Comment by sailsbot** [comment]
Thanks for posting, @BraunreutherA.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
